python/mxnet/ranking_metrics.py

```diff
@@ -0,0 +1,46 @@
+"""
+A function that implements ranking metrics
+(precision, recall, coverage and converted coverage)
+for a given position K
+"""
+
+def metrics_at_k(Y, Yhat, K):
+    """
+    Parameters
+    ----------
+    Y 	: dictionary with key = sample index and value = list of positive indices of features
+    Yhat: dict with key = sample index
+              and value = ORDERED list of indices of features, according to some score
```

Comment:
Does the model always produce ordered result? is that why you're adding the `ordered` restriction here?